wagtail_wordpress_import/test/tests/test_shortcode_handlers.py

```diff
@@ -83,7 +84,7 @@ class FooHandler(BlockShortcodeHandler):
             pass
 
         with self.assertRaises(NotImplementedError) as ctx:
-            handler = FooHandler()
+            handler = FooHandler()  # noqa
```

Comment:
**Suggestion**: Throughout this file you can remove the unused variable assignment instead of adding `# noqa`.

Reply:
Thanks Nick: Fixed